src/main/java/lotto/constant/LottoRanking.java

```diff
@@ -0,0 +1,28 @@
+package lotto.constant;
+
+import java.util.Arrays;
+
+public enum LottoRanking {
+
+    THREE(3, 5000),
```

Comment:
```suggestion
    THREE(3, 5_000),
```
언더바을 통해 가독성을 향상시킬 수 있습니다ㅎㅎ